test/integration/react-18/app/components/bar.js

```diff
@@ -0,0 +1,28 @@
+import { Suspense } from 'react'
+import dynamic from 'next/dynamic'
+import { useCachedPromise } from './promise-cache'
+
+const Foo = dynamic(() => import('./foo'), {
+  suspense: true,
+  loadableGenerated: {
+    modules: ['./foo'],
+    webpack: [require.resolveWeak('./foo')],
+  },
```

Comment:
Just curious, why is the loadable manually written here? Maybe worth adding a comment in the source.

Reply:
was trying to explicitly decalre loaded modules to make sure they're ignored with suspense, not necessary and modules should be module id with next build. deleted.